packages/editor/src/client/panel/galleryListEditPanel.tsx

```diff
@@ -85,19 +86,20 @@ export function GalleryListItem({value, onChange}: FieldProps<GalleryImageEdge>)
           removeImage={() => onChange?.({...value, image: null})}
         />
         <Form fluid={true}>
-          <FormGroup>
-            <ControlLabel>{t('blocks.imageGallery.panels.caption')}</ControlLabel>
-            <FormControl
+          <Form.Group>
+            <Form.ControlLabel>{t('blocks.imageGallery.panels.caption')}</Form.ControlLabel>
+            <Form.Control
+              name="caption"
               rows={1}
-              componentClass="textarea"
+              accepter={Textarea}
```

Comment:
Is now done via accepter and it's own component: https://rsuitejs.com/components/form/